configuration/init/systemd/tedge-mapper-aws@.service

```
@@ -0,0 +1,14 @@
+[Unit]
+Description=tedge-mapper-aws checks Thin Edge JSON measurements and forwards to AWS IoT Hub.
```

```suggestion
Description=tedge-mapper-aws checks Thin Edge JSON measurements and forwards to AWS IoT Core. 
``` 
You've only inherited this from the existing non-templated version, which was wrong.